src/main/java/utils/ApiHelper.java

```diff
@@ -0,0 +1,95 @@
+package utils;
+
+import api.ProductCode;
+import api.ProductQuantity;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import io.restassured.module.jsv.JsonSchemaValidator;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiHelper {
+    private RequestSpecification requestSpecification;
+    public Map<String, Object> storage = new HashMap<>();
+
+    public void createCart(String url) {
+        requestSpecification = RestAssured.given()
+                .baseUri(url)
+                .basePath("/api/v2/kvn/users/anonymous/carts/");
+
+        Response response = RestAssured.given()
+                .spec(requestSpecification)
+                .post();
+        String guid = response.jsonPath().get("guid");
+        storage.put("guid", guid);
+    }
+
+    // Simple RestAssured post
+    public void post(String url, String path) {
+        requestSpecification = RestAssured.given()
+                .baseUri(url)
+                .basePath(path)
+                .contentType("application/json;charset=UTF-8");
+        RestAssured.given()
+                .spec(requestSpecification)
+                .body("{\"product\": {\"code\": \"2876350\"},\"quantity\": 2}")
+                .when()
+                .post();
+    }
+
+    // Object Builder post
+    public void addProductObMap(String url) throws JsonProcessingException {
```

Comment:
test data should belong to test class and test class only. It should not be on this layer